raft/server.cc

```diff
@@ -335,6 +335,8 @@ class server_impl : public rpc_server, public server {
     }
     future<> do_on_leader_with_retries(seastar::abort_source* as, AsyncAction&& action);
 
+    future<> override_snapshot_thresholds();
+
```

Comment:
Adding a new function with only injected code seems strange. It only hides the info that the code is injected. Is there a particular reason why you did this?